Neos.ContentRepositoryRegistry/Tests/Unit/Configuration/NodeTypeEnrichmentServiceTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+declare(strict_types=1);
+namespace Neos\ContentRepositoryRegistry\Tests\Unit\Configuration;
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\ContentRepositoryRegistry\Configuration\NodeTypeEnrichmentService;
+use Neos\ContentRepositoryRegistry\Configuration\NodeTypesLoader;
+use Neos\Flow\Configuration\Source\YamlSource;
+use Neos\Flow\Core\ApplicationContext;
+use Neos\Flow\Package\FlowPackageInterface;
+use Neos\Flow\Tests\UnitTestCase;
+use Neos\Utility\Files;
+
+class NodeTypeEnrichmentServiceTest extends UnitTestCase
+{
+    private ?NodeTypeEnrichmentService $nodeTypeEnrichmentService;
+
+    private ?ApplicationContext $applicationContext;
+
+
+    public function setUp(): void
+    {
+        $this->nodeTypeEnrichmentService = new NodeTypeEnrichmentService();
+        $this->applicationContext = new ApplicationContext('Testing/SubContext');    }
+
+    private function mockPackage(string $packageKey): FlowPackageInterface
+    {
+        $mockPackage = $this->getMockBuilder(FlowPackageInterface::class)->getMock();
+        $packagePath = Files::concatenatePaths([__DIR__, 'Fixture', 'Packages', $packageKey]) . '/';
+        $mockPackage->method('getPackagePath')->willReturn($packagePath);
+        $mockPackage->method('getConfigurationPath')->willReturn(Files::concatenatePaths([$packagePath, 'Configuration']) . '/');
+        return $mockPackage;
+    }
+
+
+    /**
+     * @test
+     */
+    public function EnrichNodeTypeLabelsConfig(): void
+    {
+        $mockEnrichmentPackage = $this->mockPackage('Neos.Enrichment');
+
+        $nodeTypesLoader = new NodeTypesLoader(new YamlSource(), __DIR__);
+        $nodeConfiguration = $nodeTypesLoader->load([$mockEnrichmentPackage], $this->applicationContext);
```

Comment:
hi ;) thanks for writing tests for this!

i think this is a little to complex than it has to be :D

If we really want to use a fixture for this in a separate file, we could simply do

 `$nodeConfiguration = Yaml::parserFromFile(__DIR__ . '/my.yaml')`

buuuut separate files with part of the test fixture are imo not ideal and that doesnt scale well. Having the assertions in a complete different location makes it hard to understand the bigger picture.

I prefer having it for simple cases like this in the php file itself:

```yaml
$nodeConfiguration = YAML::parse(<<<'YAML'
'Neos.Enrichment:Translation':
  properties:
YAML);
```

this format can also be used for the `$expectedResult` and increases readability a lot.

What do you think?

Reply:
Works for me, I just thought since we are loading yaml from the file in the NodeTypesLoaderTest, I would keep the yaml origin consistent :)